connectivity/FEATURE_BLE/source/common/ble_trace_helpers.h

```diff
@@ -44,9 +43,43 @@ static inline const char* tr_as_array(T item)
     return (mbed_trace_array)((const uint8_t*)&item, sizeof(item));
 }
 
-static inline const char* to_string(UUID uuid)
+static inline char* to_string(const UUID& uuid)
 {
-    return (mbed_trace_array)(uuid.getBaseUUID(), uuid.getLen());
+    const uint8_t* buf = uuid.getBaseUUID();
+    const uint8_t  len = uuid.getLen();
+    const  size_t  max_num_row = 4;
```

Comment:
most traces only use 1, I thought there was one with 2 but I can't any right now, maybe 4 is overkill
whatever number you use please document this limitation